lib/dor/text_extraction/abbyy/file_watcher.rb

```diff
@@ -11,7 +11,7 @@ module Abbyy
       # See bin/abbyy_watcher for the script that starts this class
       # See lib/capistrano/tasks/abbyy_watcher_systemd.cap for the systemd service definition
       class FileWatcher
-        attr_reader :result_xml_path, :exceptions_path
+        attr_reader :result_xml_path, :exceptions_path, :logger, :workflow_updater
 
```

Comment:
we tend to use attr_reader instead of instance variables throughout method calls with classes